### PR TITLE
FIX: Update fluent-bit version to 2.1.3

### PR DESCRIPTION
**Pull request by MichaelBriggs-Coralogix**
FIX: Update fluent-bit version to 2.1.3
FIX: Fix kubernetes deployment

# Description

Updated fluent-bit image version to 2.1.3, updated corresponding deployments.
Clarified kubernetes (no helm) deployment documentation
Updated kubernetes (no helm) lua script to reduce errors in  pod logs

# How Has This Been Tested?

QA testing was performed with EKS cluster deployments. Pod logs were reviewed for errors and Coralogix platform was monitored for expected ingest.

# Checklist:
- [x] I have updated the relevant Helm chart(s) version(s)
- [x] I have updated the relevant component changelog(s)
- [ ] This change does not affect any particular component (e.g. it's CI or docs change)

### CODE REVIEW

File: logs/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Changelog
 
-## Fluentd-http
+## Fluentd
+
+### v1.15.2 / 2023-06-20
+
+* [DOWNGRADE] Restoring the image version to 0.0.7 in the Fluentd Helm (Coralogix Plugin) 'values.yaml' file
```
Comment (avivgold098) [inline logs/CHANGELOG.md:7]: It's realted?

File: logs/CHANGELOG.md [hunk 2]
```diff
@@ -24,6 +28,15 @@
 
 ## Fluent-Bit
 
+### v2.1.3 / 2023-06-21
+
+* [UPGRADE] Upgrade Fluent-bit version to 2.1.3
```
Comment (avivgold098) [inline logs/CHANGELOG.md:33]: There is 2.1.5, should we go with latest?

File: logs/fluent-bit/k8s-manifest/README.md [hunk 3]
```diff
@@ -28,19 +28,19 @@ Note: the configmap name is important and is being used by the daemonSet.
 
 change 'ENDPOINT' according to your logs endpoint from the table below.
 
-And apply it:
+And apply it to the monitoring namespace:
```
Comment (avivgold098) [inline logs/fluent-bit/k8s-manifest/README.md:31]: Why we mentioned is mandatory?

File: logs/fluent-bit/k8s-manifest/fluentbit-ds.yaml [hunk 4]
```diff
@@ -34,6 +34,8 @@ spec:
                 name: coralogix-keys
             - configMapRef:
                 name: fluent-bit
+            - configMapRef:
+                name: fluent-bit-env
```
Comment (avivgold098) [inline logs/fluent-bit/k8s-manifest/fluentbit-ds.yaml:38]: Who is applying this configMap?

**Review by avivgold098 — CHANGES_REQUESTED** [review]

File: logs/CHANGELOG.md (hunk 1 above)
Comment (MichaelBriggs-Coralogix) [reply logs/CHANGELOG.md:7]: This was just a cleanup of that changelog file that I noticed. It's not related to the work done in this PR, no.

Comment (avivgold098) [reply logs/CHANGELOG.md:7]: Cool, I'm resolving it

File: logs/CHANGELOG.md (hunk 2 above)
Comment (MichaelBriggs-Coralogix) [reply logs/CHANGELOG.md:33]: 2.1.3 was the latest version I could find release notes for, so we chose that one:
https://fluentbit.io/announcements/

File: logs/fluent-bit/k8s-manifest/README.md (hunk 3 above)
Comment (MichaelBriggs-Coralogix) [reply logs/fluent-bit/k8s-manifest/README.md:31]: The role binding is applied to the monitoring namespace within the RBAC manifest.

File: logs/fluent-bit/k8s-manifest/fluentbit-ds.yaml (hunk 4 above)
Comment (MichaelBriggs-Coralogix) [reply logs/fluent-bit/k8s-manifest/fluentbit-ds.yaml:38]: The README.md states they have to manually create one by that name.

File: logs/fluent-bit/image/VERSION [hunk 5]
```diff
@@ -1 +1 @@
-v0.0.3
+v2.1.3
```
Comment (avivgold098) [inline logs/fluent-bit/image/VERSION:1]: @MichaelBriggs-Coralogix sorry do we would like to switch to this versioning?

**Review by avivgold098 — CHANGES_REQUESTED** [review]

File: logs/fluent-bit/image/VERSION (hunk 5 above)
Comment (royfur) [reply logs/fluent-bit/image/VERSION:1]: Yes, that way our image version is similar to the fluentbit version just like we do in fluentd and it is less confusing to customers

**Review by avivgold098 — APPROVED** [review]

**Review by oded-dd — APPROVED** [review]
Support Logs2Metrics on the client side :)

**Pull request merged**